insights/parsers/swift_conf.py

```diff
@@ -1,8 +1,12 @@
 """
-SwiftProxyServerConf - file ``/etc/swift/proxy-server.conf``
-============================================================
+Swift Conf Files - file ``/etc/swift/``
+=======================================
 
 This module provides parsers for swift config files under /etc/swift directory.
+
+SwiftProxyServerConf - file ``/etc/swift/proxy-server.conf``
```

Comment:
By adding a line of underlines to line-7 and line-8, the documation will look better.